packages/sharing-editor/src/export/html.spec.ts

```
@@ -402,6 +402,7 @@ describe("escapeTextForJsTemplateLiteral", () => {
   ].forEach((text) => {
     it(`escapes ${text} correctly`, () => {
       const escaped = escapeTextForJsTemplateLiteral(text);
+      // oxlint-disable-next-line no-eval
       const parsed = eval("`" + escaped + "`");
```

Biome (2.4.11)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)